com.unity.render-pipelines.universal/Editor/ShaderGUI/ParticleGUI.cs

```diff
@@ -166,18 +166,10 @@ public static void FadingOptions(Material material, MaterialEditor materialEdito
             {
                 // Soft Particles
                 {
-                    EditorGUI.showMixedValue = properties.softParticlesEnabled.hasMixedValue;
-                    var enabled = properties.softParticlesEnabled.floatValue;
+                    materialEditor.ShaderProperty(properties.softParticlesEnabled, Styles.softParticlesEnabled);
+                    var enabled = properties.softParticlesEnabled.floatValue >= 0.5f;
```

Comment:
Personally I don't think >= is needed. 0 == false, true otherwise is well defined.